source/MapPanel.cpp

```diff
@@ -1143,16 +1145,21 @@ void MapPanel::DrawTooltips()
 
 
 
-void MapPanel::DrawPointer(const System *system, Angle &angle, const Color &color, bool bigger)
+void MapPanel::DrawPointer(const System *system, int &systemCount, const Color &color, bool bigger)
 {
-	DrawPointer(Zoom() * (system->Position() + center), angle, color, true, bigger);
+	DrawPointer(Zoom() * (system->Position() + center), systemCount, color, true, bigger);
 }
 
 
 
-void MapPanel::DrawPointer(Point position, Angle &angle, const Color &color, bool drawBack, bool bigger)
+void MapPanel::DrawPointer(Point position, int &systemCount, const Color &color, bool drawBack, bool bigger)
 {
-	angle += Angle(30.);
+	Angle angle = Angle();
+	++systemCount;
+	angle += Angle(30.*systemCount);
```

Comment:
Minor detail, but I think it's possible to construct the Angle in one go:
`Angle angle = Angle(30.*systemCount);`
Also for the special system above.